src/preferences/dialog/dlgprefmixerdlg.ui

```diff
@@ -162,7 +163,7 @@
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
-          <property name="fixedSize" stdset="0">
+          <property name="fixedSize">
```

Comment:
just out of curiosity, any idea what `stdset="0"` does? I couldn't find anything online.

Reply:
Nope, I didn't do a research. Tbh I'm a tired of debugging Qt quirks, crawling Qt forums, and I only have 6.2.3 to test with. So as long as it looks as intended on all platforms :man_shrugging:

Reply:
no worries.